### PR TITLE
Minor refactorings

### PR DESCRIPTION
**Pull request by WhiteBlackGoose**

### CODE REVIEW

File: .github/workflows/Build.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      with:
+        submodules: 'recursive'
+    - name: Setup .NET 5
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '5.0.100-rc.1.20452.10'
+    - name: Build
+      run: dotnet build AngouriMath
+  Everything:
+    runs-on: windows-latest  # Window application cannot be run on other OS
```
Comment (Happypig375) [inline .github/workflows/Build.yml:21]:
```suggestion
    runs-on: windows-latest  # Windows application cannot be run on other OS
```

File: AngouriMath/Functions/Boolean/Simplifier.cs [hunk 2]
```diff
@@ -7,33 +7,60 @@
...
 
 namespace AngouriMath
 {
     partial record Entity
     {
-        // Theoretically, we might've united And and Or under Binary operator
-        // and avoid duplication
+        // ngl; it looks awful
+
+        protected interface IBranchGetter<T>
```
Comment (Happypig375) [inline AngouriMath/Functions/Boolean/Simplifier.cs:20]: `IBinaryNode`?

File: .github/workflows/Build.yml (hunk 1 above)
Comment (WhiteBlackGoose) [reply .github/workflows/Build.yml:21]: That's super minor tbh

Comment (Happypig375) [reply .github/workflows/Build.yml:21]: What is a Window application? You apply a window on some other object?

File: AngouriMath/Functions/Evaluation/Patterns/Patterns.EqualityInequality.cs [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+{
+    internal static partial class Patterns
+    {
+        private static bool IsRealPositive(Entity entity)
+            => entity is Real re && re > 0;
+
+        private static bool IsZero(Entity entity)
+            => entity is Real re && Real.IsZero(re);
+
+        // Suggestions to refactor this?
```
Comment (Happypig375) [inline AngouriMath/Functions/Evaluation/Patterns/Patterns.EqualityInequality.cs:29]: Use a `switch`

**Pull request merged**